README.md

```diff
@@ -16,6 +16,30 @@ Here can the Firebase functions be found that are necessary to successfully setu
 * Login to Firebase by `firebase login` and choose your project.
 * Run `firebase deploy` from the project root.
 
+### Value Investment KPIs
+
+## Company Analysis
+
+
+##
+
+
+## Financials Input required annually for Big Five (Last 10 Years)
+
+|Name (DE)|Source|Name (EN)|Finacial Modeling Prep API| Comments |
+|---|---|---|---|---|
+|Summe langfristiges Fremdkapital| Bilanz (Passiva)  | Balance Sheet > Long-term Debt  | https://financialmodelingprep.com/api/v3/financials/balance-sheet-statement/AAPL  |   |
+|Summe Eigenkapital   | Bilanz (Passiva)  | Balance Sheet > Total shareholders equity  |   |   |
```

Comment:
Also found this here: https://financialmodelingprep.com/api/v3/financials/balance-sheet-statement/AAPL